internal/kafka/test/integration/serviceaccounts_test.go

```diff
@@ -270,6 +270,21 @@ func TestServiceAccounts_InputValidation(t *testing.T) {
 	Expect(err).Should(HaveOccurred())
 }
 
+func TestServiceAccounts_SsoProvider(t *testing.T)  {
+	ocmServer := mocks.NewMockConfigurableServerBuilder().Build()
+	defer ocmServer.Close()
+
+	h, client, teardown := test.NewKafkaHelper(t, ocmServer)
+	defer teardown()
+
+	account := h.NewRandAccount()
+	ctx := h.NewAuthenticatedContext(account, nil)
+
+	_,resp, err := client.SecurityApi.GetSsoProvider(ctx)
+	Expect(err).ShouldNot(HaveOccurred())
+	Expect(resp.StatusCode).To(Equal(http.StatusOK))
```

Comment:
We could aslo check against the different fields in the sso_provider response body